c2corg_ui/static/js/feed.js

```diff
@@ -0,0 +1,153 @@
+goog.provide('app.FeedController');
+goog.provide('app.feedDirective');
+
+goog.require('app');
+goog.require('app.Api');
+goog.require('app.Authentication');
+goog.require('app.utils');
+
+
+/**
+ * @return {angular.Directive} The directive specs.
+ */
+app.feedDirective = function() {
+  return {
+    restrict: 'A',
+    controller: 'appFeedController as feedCtrl'
+  };
+};
+app.module.directive('appFeed', app.feedDirective);
+
+
+/**
+ * @param {angular.Scope} $scope Scope.
+ * @param {angular.Attributes} $attrs Attributes.
+ * @param {app.Authentication} appAuthentication
+ * @param {app.Api} appApi Api service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping
+ * @constructor
+ * @ngInject
+ * @export
+ */
+app.FeedController = function($scope, $attrs, appAuthentication, appApi, $sce) {
+
+  /**
+   * @type {angular.Scope}
+   * @private
+   */
+  this.scope_ = $scope;
+
+  /**
+   * @type {app.Api}
+   * @private
+   */
+  this.api_ = appApi;
+
+  /**
+   * @type {app.Authentication}
+   * @private
+   */
+  this.auth_ = appAuthentication;
+
+  /**
+   * @type {angular.$sce} $sce Angular Strict Contextual Escaping
+   * @private
+   */
+  this.sce_ = $sce;
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.location_ = $attrs['location'];
```

Comment:
That would be more consistent to:
* name the attribute according to the directive, eg. ``appFeedLocation``
* retrieve this param by making the constructor a ``@struct`` and pass the argument in the directive's scope or in ``bindToController`` (as in https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/imageuploader.js)